cmd/kyma/provision/gcp/cmd.go

```diff
@@ -0,0 +1,122 @@
+package gcp
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"strings"
+
+	"github.com/kyma-project/cli/internal/kube"
+
+	hf "github.com/kyma-incubator/hydroform"
+	"github.com/kyma-incubator/hydroform/types"
+	"github.com/kyma-project/cli/internal/cli"
+	"github.com/kyma-project/cli/internal/files"
+	"github.com/spf13/cobra"
+)
+
+type command struct {
+	opts *Options
+	cli.Command
+}
+
+//NewCmd creates a new minikube command
+func NewCmd(o *Options) *cobra.Command {
+	c := command{
+		Command: cli.Command{Options: o.Options},
+		opts:    o,
+	}
+
+	cmd := &cobra.Command{
+		Use:   "gcp",
+		Short: "Provisions a GCP cluster.",
+		Long:  `Use this command to provision GCP for Kyma installation.`,
+		RunE:  func(_ *cobra.Command, _ []string) error { return c.Run() },
+	}
+
+	cmd.Flags().StringVarP(&o.Name, "name", "n", "", "Name of the cluster to provision.")
+	cmd.Flags().StringVarP(&o.Project, "project", "p", "", "Name of the GCP Project where to provision the cluster in.")
+	cmd.Flags().StringVarP(&o.CredentialsFile, "credentials", "c", "", "Path to the GCP service account key file.")
+	cmd.Flags().StringVarP(&o.KubernetesVersion, "kube-version", "k", "1.14.6", "Kubernetes version of the cluster to provision.")
+	cmd.Flags().StringVarP(&o.Location, "location", "l", "europe-west3-a", "Location of the cluster to provision.")
+	cmd.Flags().StringVarP(&o.MachineType, "type", "t", "n1-standard-4", "Type of machine of the cluster to provision.")
+	cmd.Flags().IntVar(&o.DiskSizeGB, "disk-size", 30, "Specifies the disk size in GB of the cluster to provision.")
+	cmd.Flags().IntVar(&o.CPUS, "cpus", 4, "Specifies the number of CPUs of the cluster to provision.")
```

Comment:
Does this still have an effect when we specify machine-type?

Reply:
Actually, hydroform does not use this property in any provider at the moment.
We might have this in the future though.

Maybe we can just remove the flag completely from the CLI until there is an actual use case for it, WDYT?

Reply:
I think it'd be better if we just remove the flag. Otherwise, it would be misleading for the users when they specify the number of CPUs and see that it is not taken into account.

Reply:
done thanks!